jujugui/static/gui/src/app/models/models.js

```diff
@@ -607,7 +607,17 @@ YUI.add('juju-models', function(Y) {
         @type {Object}
         @default undefined
       */
-      activePlan: {}
+      activePlan: {},
+
+      /**
+        In a multi-series charm the service of the service will not always
```

Comment:
the service of the service?
Please make this comment less confusing: what's stored here, and why?